### ISSUE TITLE
Missing "NumberedRecurrence" in Exchange EWS SOAP

### ISSUE THREAD

**Issue by Jisco** [mguessan/davmail#373]
Hi,
I have a problem with recurring appointments that have a certain count.

These are created correctly, but after an update via soap they become "infinite".

The problem seems to be that the count of appointments is not taken into account (via [NumberedRecurrence](https://learn.microsoft.com/en-us/exchange/client-developer/web-service-reference/numberedrecurrence?redirectedfrom=MSDN)) at all and is set [here](https://github.com/mguessan/davmail/blob/2938a7bc37c280fe072c235914bb8e644d9a18d6/src/java/davmail/exchange/ews/RecurrenceFieldUpdate.java#L126).

Or have I missed something?

**Comment by mguessan** [comment]
That's correct, there are many unusual recurrence patterns that are not implemented.

Tried a quick fix in trunk, can you please check ?

**Comment by ifrh** [comment]
@mguessan  wrote :
> Tried a quick fix in trunk, can you please check ?

I tried it, too. Works for me.

**Comment by esabol** [comment]
Sounds like this issue can be closed. 👍

**Comment by mguessan** [comment]
Thanks for your feedback, closing issue
